webapp/javascript/ui/Tooltip.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */
+import React from 'react';
+import styles from './Tooltip.module.scss';
+
+const Tooltip = ({
+  syncEnabled,
+  visible,
+}: {
+  syncEnabled: string | boolean;
+  visible: boolean;
+}) => {
+  return (
+    <div
+      onClick={(e) => e.stopPropagation()}
+      className={`${visible ? styles.visible : ''} ${
+        syncEnabled ? styles.tooltipSyncEnabled : styles.tooltip
+      }`}
+      role="tooltip"
+    >
+      {syncEnabled ? 'Unsync search bars' : 'Sync search bars'}
```

Comment:
I don't get this, when I said a standalone component I was referring to something that can stand on its own. Like a `button`. Here this component would NEVER be used on its own, since its text refers to sync.

Reply:
@eh-am, sorry for such a confusion, made a universal standalone Tooltip.